twingate/internal/client/converter.go

```diff
@@ -230,3 +230,31 @@ func (r ResourceNode) ToModel() *model.Resource {
 func (q readResourcesByNameQuery) ToModel() []*model.Resource {
 	return q.Resources.ToModel()
 }
+
```

Comment:
Its something I missed in the restructure review but Im not sure I understand the purpose of `converter.go` - we define `createServiceAccountQuery` in `service-account.go` why cant its methods be there?  
(same for all the other structs that their methods are here)

Reply:
here in `converter.go` I've collected simple utilities functions which converts queries objects to models

I think in `service-account.go` would be better to have only code related to API request, and this object conversions like lower layer details

I like the idea with separate package for `queries`, I think this can help organise code better :)

Reply:
ok so lets merge this and refactor to `queries` on a separate PR